File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/formorder/AdrAndEmail.java
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.formorder;

//Необходим при заказе
//для Полей емейл и адрес;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AdrAndEmail {

    @SerializedName("fld2")
    @Expose
    String fld2;

    @SerializedName("fld3")
    @Expose
    String fld3;

    public AdrAndEmail(String fld2, String fld3){
        this.fld2 = fld2;
        this.fld3 = fld3;
    }
}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/formorder/AdrAndEmail.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.formorder

//Необходим при заказе
//для Полей емейл и адрес;

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AdrAndEmail(@field:SerializedName("fld2")
                  @field:Expose
                  internal var fld2: String, @field:SerializedName("fld3")
                  @field:Expose
                  internal var fld3: String)
